taichi/ir/type_utils.cpp

```diff
@@ -56,6 +56,11 @@ int data_type_size(DataType t) {
   //  setting a loud failure on pointers);
   //  2. Support pointer types here.
   t.set_is_pointer(false);
+  // If DataType is a TensorType, returns its element type size
```

Comment:
Why do we make the size of `TensorType` its element type size?

Reply:
fix